### ISSUE TITLE
[RPC] should not return "error": null in JSON-RPC responses

### ISSUE THREAD

**Issue by NodeGuy** [eris-ltd/eris-db#270]
This request:

``` shell
curl --data '{"jsonrpc":"2.0","method":"erisdb.getChainId","params":null,"id":"23"}' --header "Content-Type:application/json" http://192.168.99.100:33367/rpc
```

responds with:

```
{"result":{"chain_id":"blockchain"},"error":null,"id":"23","jsonrpc":"2.0"}
```

but the [JSON-RPC specification](http://www.jsonrpc.org/specification#response_object) states:

> error
> This member MUST NOT exist if there was no error triggered during invocation.

**Comment by silasdavis** [comment]
Will be superseded by RPC work

**Comment by benjaminbollen** [comment]
This should not have been closed; rather should now be addressed in #465
